packages/uniforms-bridge-simple-schema/__tests__/SimpleSchemaBridge.ts

```diff
@@ -80,9 +82,9 @@ describe('SimpleSchemaBridge', () => {
         throw 'ValidationError';
       };
     },
-  };
+  } as unknown as SimpleSchema;
```

Comment:
Why do we need an assertion here?

Reply:
Because a real SimpleSchema instance has many more fields.